app/scripts/components/LocationMap.js

```diff
@@ -49,6 +51,62 @@ class LocationMap extends Component {
   }
 
   render() {
+    const MyPopup = ({ events, place }) => {
```

Comment:
`MyPopup` is a relatively generic name, I would suggest something more related to how we actually use this in this context. `EventInfoPopup` maybe? Similar thing with the other component names starting with `My..`

Reply:
That's what you get for following tutorial language too closely....